src/views/Project/issues.graphql

```diff
@@ -0,0 +1,26 @@
+query Issues($searchQuery: String!) {
+  search(first: 100, type: ISSUE, query: $searchQuery) {
+    nodes {
+      ... on Issue {
+        title
+        updatedAt
+        number
+        repository {
+          name
+        }
+        url
+        labels(last:5) {
+          nodes {
+            name
+            color
+          }
+        }
+        assignees(last:1) {
+          nodes {
+            login
+          }
+        }
+      }
+    }
+  }
```

Comment:
Do we need all of that data? If there's anything unused (e.g., color), feel free to remove it from the query.

Reply:
i plan to use chips for the tag later, but idk if i can use the color directly since bugzilla has no color, should i remove it for now?

Reply:
Yes remove it for now and we can add it later. The beauty of graphQL is that we can reduce bandwidth by only asking for what we want :)

Reply:
ok done